### DISCUSSION TITLE
GitHub Copilot now available for Visual Studio 2022... but no

### DISCUSSION THREAD

**Discussion by uploaded** [community/community#13854] [Copilot Conversations] [answered]
Greetings,

Today I got out of bed with great news:

"GitHub Copilot is now available from Visual Studio 2022 for everyone in the technical preview."

But sadly this isn't true. The usual message keeps showing up "You don't have access to GitHub Copilot. "

I've been waiting for a lot of months and still I can't get to access a preview version, even when its for "everyone".

Is there any updates on your end ? 

Thanks for the feedback!
uploaded
![copilot](https://user-images.githubusercontent.com/1611689/160794315-d17de676-ea66-4e0e-b4ab-9abcbdb9d4a5.png)

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
